src/include/pg_version_compat.h

```diff
@@ -57,6 +57,9 @@ typedef int ObjectClass;
 #define OCLASS_AM AccessMethodRelationId
 #define OCLASS_TSTEMPLATE TSTemplateRelationId
 
+#define Anum_pg_collation_colliculocale Anum_pg_collation_colllocale
+#define Anum_pg_database_daticulocale Anum_pg_database_datlocale
+
```

Comment:
Variable names throughout the codebase referring to older names such as `colliculocale` could be fixed to match the renaming. This is if want to be more compatible with PG17 rather than older versions.

Reply:
That is a good idea since we essentially are moving towards newer PG versions, and at some point, variables named "...iculocale" will not make sense. I will rename them.